packages/app-store/riverside/config.json

```diff
@@ -3,8 +3,8 @@
   "name": "Riverside",
   "slug": "riverside",
   "type": "riverside_video",
-  "imageSrc": "/api/app-store/riverside/icon.svg",
-  "logo": "/api/app-store/riverside/icon.svg",
+  "imageSrc": "/api/app-store/riverside/icon-dark.svg",
```

Comment:
We've decided to add "dark" to a few icons which are mainly of a dark color. In the render of the OG images we check this name and don't render a gradient blob in the background if it's a dark image.